var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -743,6 +752,21 @@ def define_enable(suffix, value=None):
                 ]
             )
 
+        if "+pytrilinos2" in spec:
+            options.append(define_trilinos_enable("PyTrilinos2", True))
+            if "+test" in spec:
+                options.append(define("PyTrilinos2_ENABLE_TESTS", True))
```

Comment:
Slightly safer and shorter:
```suggestion
           options.append(define_from_variant("PyTrilinos2_ENABLE_TESTS", "test"))
```